erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -183,6 +190,18 @@ bool WebRtcConnection::setRemoteSdpInfo(std::shared_ptr<SdpInfo> sdp) {
 
 std::shared_ptr<SdpInfo> WebRtcConnection::getLocalSdpInfo() {
   ELOG_DEBUG("%s message: getting local SDPInfo", toLog());
+  forEachMediaStream([this] (const std::shared_ptr<MediaStream> &media_stream) {
```

Comment:
I don't get why this is added here... Aren't we adding the SinkSSRCs to the local_sdp in both `createOffer` and `processRemoteSdp`. So in both cases of the negotiation we should have the local_sdp correctly filled in.

Reply:
agree, I'll make sure we don't need this by running some test